source/features/file-finder-buffer.tsx

```diff
@@ -25,11 +26,10 @@ function pjaxStartHandler(event: CustomEvent): void {
 	const repoName = select('.pagehead h1 strong, [itemprop="name"]')!;
 	repoName.classList.remove('mr-2');
 	repoName.after(
-		<span className="path-divider flex-self-stretch">/</span>,
+		<span className="mx-1 flex-self-stretch">/</span>,
 		<span className="flex-self-stretch mr-2">{bufferField}</span>
 	);
 	bufferField.focus();
-	select('.pagehead-actions')!.remove();
```

Comment:
I didn't bother to find why this is necessary, correct me if I'm wrong

Reply:
Old layout

Reply:
The feature cleans the whole line to avoid wrapping. When the field appears, ci-link, OctoTree’s icon and the actions should all disappear.